velox/dwio/common/Options.h

```diff
@@ -640,6 +640,11 @@ struct WriterOptions {
   std::optional<uint8_t> parquetWriteTimestampUnit;
   std::optional<uint8_t> zlibCompressionLevel;
   std::optional<uint8_t> zstdCompressionLevel;
+  std::map<std::string, std::string> configs;
```

Comment:
What is this for? That seems to be a catch all and I don't really see it used.

Reply:
Removing this since it is not used. I was looking at getDwrfOptions() in /velox/dwio/dwrf/writer/Writer.cpp and it had `std::map<std::string, std::string> configs;` so I was thinking of maybe passing it as part of the common writerOptions but I decided not to go this route